### ISSUE TITLE
[stable/gcloud-sqlproxy] targetPort instance name

### ISSUE THREAD

**Issue by simowaer** [helm/charts#4421]
**Is this a request for help?**: NO

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

**Version of Helm and Kubernetes**: 2.8

**Which chart**: gcloud-sqlproxy

**What happened**: spec.ports[0].targetPort must be no more than 15 characters, my instance has more than 15 characters

**What you expected to happen**: the targetPort should be the name of my instance

**How to reproduce it** (as minimally and precisely as possible): create a cloudsql with more than 15 characters and add it as the instance in values.yaml

**Anything else we need to know**:

**Comment by 27Bslash6** [comment]
Same issue here. 

CloudSQL permits instance names longer than 15 characters, this chart or the underlying proxy software should permit that.

Helm 2.9.0
Kubernetes 1.9.7-gke.0

**Comment by klaus** [comment]
https://kubernetes.io/docs/reference/federation/#_v1_containerport states the name must be an IANA_SVC_NAME which will be difficult when using ".instance". I just switched it to an enumerated port-name and that's working for me (but only one instance, ...)
```
        {{- range $i, $v := .Values.cloudsql.instances }}
        - name: sql-{{ $i }}
          containerPort: {{ $v.port }}
        {{- end }}
```

**Comment by trnubo** [comment]
It looks like this issue was fixed in PR #5664 but v0.3.4 of the chart doesn't seem to be available in the stable kubernetes-charts repo yet. How long does it normally take before a release is available in the repo?
